examples/testapp/src/components/SDKConfig/SDKConfig.tsx

```diff
@@ -188,7 +188,7 @@ export function SDKConfig() {
                   required
                   onChange={(e) =>
                     handleOnrampPrefill("chainId")({
-                      target: { value: parseInt(e.target.value, 10) },
+                      target: { value: Number.parseInt(e.target.value, 10) },
```

Comment:
Whats the prompt for this change? Just a style preference? If so is the an eslint rule to enforce it?

Reply:
There's no prompt per-se, and there's no functional change. Its just a small update to get in line with es6's modularization of globals. Good call on the eslint rule to enforce this - will add that & resolve conflicts today